gate-teamware/templates/deployment-backend.yaml

```diff
@@ -132,9 +132,39 @@ spec:
                 key: refresh-token
           {{- end }}{{/* if backend gmail */}}
           {{- end }}{{/* with .Values.email */}}
+
+          {{- with $.Values.privacyPolicy }}
+          # Privacy Policy Settings
+          {{- with .host.name }}
+          - name: PP_HOST_NAME
+            value: {{ . | quote }}
+          {{- end }}
+          {{- with .host.address }}
+          - name: PP_HOST_ADDRESS
+            value: {{ . | quote }}
+          {{- end }}
+          {{- with .host.contact }}
+          - name: PP_HOST_CONTACT
+            value: {{ . | quote }}
+          {{- end }}
+          {{- with .admin.name }}
```

Comment:
@davidwilby Would it be helpful to have the `admin.*` options default to the corresponding `host.*` ones if there's no explicit admin option set?  This would simplify the (common) case where the host and admin are the same person.  We could do this by saying:

```
{{- with (coalesce .admin.name .host.name) }}
```

(`coalesce` takes a sequence of values and returns the first non-empty value, or `nil` if they're all empty, and `with` skips its body if asked to operate on something that is `nil` or empty)

Reply:
> I've cleaned up the template a little with my own commits, and left a couple of comments with specific questions.

Thanks for this!

> Recommend we don't actually merge this PR until the teamware PR is finished, as there will be more changes to make if we want to support bind-mounting a custom policy.

Agreed.




> @davidwilby Would it be helpful to have the `admin.*` options default to the corresponding `host.*` ones if there's no explicit admin option set? This would simplify the (common) case where the host and admin are the same person. We could do this by saying:
> 
> ```
> {{- with (coalesce .admin.name .host.name) }}
> ```
> 
> (`coalesce` takes a sequence of values and returns the first non-empty value, or `nil` if they're all empty, and `with` skips its body if asked to operate on something that is `nil` or empty)

Yes, this sounds good, would save some replication.